Makefile

```diff
@@ -1,6 +1,3 @@
-include .env
```

Comment:
I restored it, but you don't have to manually do it either. You can just `source` your env file before starting to develop.